test/scalarstats.jl

```diff
@@ -332,30 +332,39 @@ s = summarystats(1:5)
 @test isa(s, StatsBase.SummaryStats)
 @test s.min == 1.0
 @test s.max == 5.0
+@test s.nobs   == 5
+@test s.nmiss  == 0
 @test s.mean   ≈ 3.0
 @test s.median ≈ 3.0
 @test s.q25    ≈ 2.0
 @test s.q75    ≈ 4.0
+@test s.sd     ≈ 1.5 atol=1e-1
```

Comment:
Use a lower tolerance? I guess you should be able to keep the default tolerance if you copy the actual value.

Reply:
Yeah, it's updated now...